python/ql/src/meta/alerts/RequestHandlers.ql

```diff
@@ -0,0 +1,23 @@
+/**
+ * @name Request Handlers
+ * @description HTTP Server Request Handlers
+ * @kind problem
+ * @problem.severity recommendation
+ * @id py/meta/alerts/request-handlers
+ * @tags meta
+ * @precision very-low
```

Comment:
Is the precision really that bad? Or is this just a way of ensuring that this query doesn't run automatically?

Reply:
I just assumed this was a way of ensuring that this query doesn't run automatically, when I copied this from JS

Reply:
Aha! I didn't know there was a precedent. In that case, never mind. :slightly_smiling_face: